record/program/src/instruction.rs

```diff
@@ -0,0 +1,176 @@
+//! Program instructions
+
+use crate::id;
+use borsh::{BorshDeserialize, BorshSerialize};
+use solana_program::{
+    instruction::{AccountMeta, Instruction},
+    pubkey::Pubkey,
+    sysvar,
+};
+
+/// Instructions supported by the program
+#[derive(Clone, Debug, BorshSerialize, BorshDeserialize, PartialEq)]
+pub enum RecordInstruction {
+    /// Create a new document
+    ///
+    /// Accounts expected by this instruction:
+    ///
+    /// 0. `[writable]` Data account, must be uninitialized
+    /// 1. `[]` Document authority
```

Comment:
"Document" and "Document authority" are used in a couple places.  If we're going with Record then I think these should be updated to match.

Reply:
Done in 0d65c8d